packages/aila/src/protocol/schema.ts

```diff
@@ -117,392 +115,312 @@ export const QuizOptionalSchema = z.array(QuizQuestionOptionalSchema);
+    with 'I can' and outline what pupils should be able to know/and or be able to do by the end of 
+    the lesson. Written in age appropriate language. May include a command word. Written in the PUPIL 
+    voice. ${minMaxText({ max: 190, entity: "characters" })}`,
+  learningCycles: dedent`An array of learning cycle outcomes. Should include a command word. Should 
+    be succinct. Should outline what pupils should be able to do/understand/know by the end of the 
+    learning cycle. Written in the TEACHER_TO_PUPIL_SLIDES voice. ${minMaxText({
+      min: 1,
+      max: 3,
+      entity: "elements",
+    })}`,
+  priorKnowledge: dedent`An array of prior knowledge statements, each being a succinct sentence. 
+    Written in the EXPERT_TEACHER voice. ${minMaxText({
+      min: 1,
+      max: 5,
+      entity: "elements",
+    })}`,
+  keyLearningPoints: dedent`An array of learning points, each being a succinct sentence. ${minMaxText(
+    {
+      min: 3,
+      max: 5,
+      entity: "elements",
+    },
+  )}`,
+  starterQuiz: dedent`The starter quiz for the lesson, which tests prior knowledge only, ignoring the content that is 
+    delivered in the lesson. Obey the rules as specified in the STARTER QUIZ section of the lesson plan 
+    guidance. Written in the TEACHER_TO_PUPIL_SLIDES voice.`,
+  exitQuiz: dedent`The exit quiz for the lesson, which tests the content that is delivered in the lesson. Written 
+    in the TEACHER_TO_PUPIL_SLIDES voice.`,
+  additionalMaterials:
+    "Any additional materials or notes that are required or useful for the lesson",
+} as const;
+
+export const LessonTitleSchema = z
+  .string()
+  .describe(LESSON_PLAN_DESCRIPTIONS.title);
+
+export const KeyStageSchema = z
+  .union([
+    z.enum([
+      "key-stage-1",
+      "key-stage-2",
+      "key-stage-3",
+      "key-stage-4",
+      "key-stage-5",
+      "early-years-foundation-stage",
+      "specialist",
+      "further-education",
+      "higher-education",
+    ]),
+    z.string(),
+  ])
+  .describe(LESSON_PLAN_DESCRIPTIONS.keyStage);
+
+export const SubjectSchema = z
+  .string()
+  .describe(LESSON_PLAN_DESCRIPTIONS.subject);
+
+export const TopicSchema = z.string().describe(LESSON_PLAN_DESCRIPTIONS.topic);
+
+export const LearningOutcomeSchema = z
+  .string()
+  .describe(LESSON_PLAN_DESCRIPTIONS.learningOutcome);
+
+export const LearningCyclesSchema = z
+  .array(z.string())
+  .describe(LESSON_PLAN_DESCRIPTIONS.learningCycles);
+
+export const PriorKnowledgeSchema = z
+  .array(z.string())
+  .describe(LESSON_PLAN_DESCRIPTIONS.priorKnowledge);
+
+export const KeyLearningPointsSchema = z
+  .array(z.string())
+  .describe(LESSON_PLAN_DESCRIPTIONS.keyLearningPoints);
+
+export const AdditionalMaterialsSchema = z
+  .string()
+  .optional()
+  .describe(LESSON_PLAN_DESCRIPTIONS.additionalMaterials);
+
+// Main schema
 export const CompletedLessonPlanSchema = z.object({
```

Comment:
The overall lesson plan schema now has per-section schema definitions including descriptions